lib/nerves_ssh/options.ex

```diff
@@ -289,4 +294,21 @@ defmodule NervesSSH.Options do
   else
     defp decode_key(key), do: :public_key.ssh_decode(key, :auth_keys)
   end
+
+  defp generate_system_keys!() do
+    File.mkdir_p!(@default_device_system_dir)
+    key = X509.PrivateKey.new_ec(:ed25519)
```

Comment:
```suggestion
    key = :public_key.generate_key({:namedCurve, :ed25519})
```
I like `x509`, but I'm wondering if the Erlang functions might be simple enough to avoid the extra dependency.